HPCC-15564 Document DESDL Front End

g-pan opened the pull request:
Fix HPCC-15564 Document DESDL Front End
Document new DESDL front end functionality in the ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>
@JamesDeFabia (optionally) please review if you can
@rpastrana please review

g-pan committed:
HPCC-15564 Document DESDL Front End

Fix HPCC-15564 Document DESDL Front End
Document new DESDL front end functionality in the ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest
Sha: c891cf474925116e0600dd49246ff78c02753d09
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.757 seconds |

Install hpccsystems-platform-community_6.2.0-2.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK

richardkchapman commented:
I changed the target without any need for rebasing. Reviewers, please proceed

g-pan commented:
thank you

JamesDeFabia commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+          binding.</para>
+
+          <para>Press the <emphasis role="bold">Configuration</emphasis>
+          button to view, or add to the configurations. <figure>
```
button to view, or add to the configurations. <--remove  comma

JamesDeFabia commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+      role="bold">Dynamic ESDL</emphasis> from the navigation sub-menu bar.
+      </para>
+
+      <para>The DESDL based ESP services available are listed in the navigator
```
DESDL-based (needs hyphen)

JamesDeFabia reviewed (changes requested):
Other than my 2 comments, this looks good from my POV, but please wait for the SME to approve, too.

g-pan committed:
HPCC-15564 Document DESDL Front End

Fix HPCC-15564 Document DESDL Front End
Document new DESDL front end functionality in the ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest
Sha: 63525918b5a9cfb162996f4bb38215b7da0bb9ee
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.507 seconds |

Install hpccsystems-platform-community_6.2.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK

richardkchapman commented:
@rpastrana Please review

rpastrana commented:
@g-pan send me the latest PDF based on these changes, thanks.

rpastrana commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+      ESDL Binding information.. For more information about Dynamic ESDL refer
+      to the documentation; <emphasis>Dynamic ESDL</emphasis> available from
+      the HPCC Systems portal: <ulink
+      url="https://hpccsystems.com/download/documentation/all">https://hpccsystems.com/download/documentation/all</ulink></para>
```
This link should point to the actual DESDL Doc

rpastrana commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+      <sect2 id="USING_Dynamic_ESDL">
+        <title>Using Dynamic ESDL</title>
+
+        <para>Click on the triangle to expand the ESP process and display the
```
rather than "triangle" we might want to say "expand/collapse icon"

rpastrana commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+          binding.</para>
+
+          <para>Press the <emphasis role="bold">Configuration</emphasis>
+          button to view or add to the configurations. <figure>
```
"to view or edit the method configurations."

rpastrana commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+
+          <para><orderedlist>
+              <listitem>
+                <para>press the arrow next to the method (if there are are
```
There's a couple of issues w/ this line: Press should be capitalized, and there seems to be an extra "are" in the parenthesis, but I recommend changing this sentence to something like:
"Expand the target method configuration attributes (if there are.."

rpastrana commented on docs/ECLWatch/TheECLWatchMan.xml:
```diff
@@ -1138,6 +1138,188 @@
+              </mediaobject>
+            </figure></para>
+
+          <para><emphasis role="bold">To Add an attribute:</emphasis></para>
```
It feels like we should explain that the methods are configured by adding/editing these attributes.

rpastrana reviewed (changes requested):
@g-pan left several comments.

g-pan committed:
HPCC-15564 Document DESDL Front End

Fix HPCC-15564 Document DESDL Front End
Document new DESDL front end functionality in the ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

rpastrana commented:
@g-pan PDF reviewed, looks fine.

HPCCSmoketest commented:
Automated Smoketest
Sha: 9a093b94e127a47f5ca8a906fd6e5b27bad0cd70
Build: failed
Number of Error(s): 8
\r  3 2573k    3 81927    0     0   101k      0  0:00:25 --:--:--  0:00:25  101kError: no ID for constraint linkend: ssl4esp.
/bin/ar: lib/libcouchbaseS.a: Error reading generated/probes.o: File truncated
gmake[5]: *** [lib/libcouchbaseS.a] Error 1
gmake[4]: *** [CMakeFiles/couchbaseS.dir/all] Error 2
gmake[3]: *** [all] Error 2
make[2]: *** [plugins/couchbase/libcouchbase-build/lib/libcouchbase.so] Error 2
make[1]: *** [plugins/couchbase/CMakeFiles/libcouchbase-build.dir/all] Error 2
make: *** [all] Error 2

rpastrana commented:
@AttilaVamos  the changes that went in this PR are documentation related, not sure how it could have caused the smoketest error reported, can you look into this? Thanks.

HPCCSmoketest commented:
Automated Smoketest
Sha: 9a093b94e127a47f5ca8a906fd6e5b27bad0cd70
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.9 seconds |

Install hpccsystems-platform-community_6.2.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK
